tests/e2e/pageObjects/slideViewerPage.js

```diff
@@ -0,0 +1,41 @@
+const { config } = require('../cucumber.conf')
+
+class SlideViewerPage {
+  constructor() {
+    this.homePageSelector = '.oc-login-bg'
+    this.loginFormSelector = '.oc-login-form'
+    this.usernameInputFieldSelector = '#oc-login-username'
+    this.passwordInputFieldSelector = '#oc-login-password'
+    this.loginBtnSelector = '.jss8 .oc-button-primary'
+    this.filesContainerSelector = '#files-view'
+    this.contextMenuBtnSelector = '.resource-table-btn-action-dropdown'
+    this.openInPresentationViewerBtnSelector = '.oc-files-actions-presentation-viewer-trigger'
+    this.slidesContainerSelector = '#presentation-viewer-main .slides'
+    this.previousSlideBtnSelector = '.navigate-left'
+    this.nextSlideBtnEnabledSelector = 'button.navigate-right.enabled'
+    this.slideViewerHomepageSelector = '#presentation-viewer-main'
+    this.visibleSectionSelector = 'section:not([hidden])'
+    this.signInPageUrl = config.baseUrlOcis
+  }
+
+  async navigate() {
+    await page.goto(this.signInPageUrl)
+  }
+
+  async login(username, password) {
+    await page.fill(this.usernameInputFieldSelector, username)
+    await page.fill(this.passwordInputFieldSelector, password)
+    await page.click(this.loginBtnSelector)
+  }
+
+  async openMDFileInPresentationViewer() {
+    await page.click(this.contextMenuBtnSelector)
+    await page.click(this.openInPresentationViewerBtnSelector)
+  }
+
+  async clickNextSlideBtnEnabled() {
```

Comment:
I'm not sure about the naming, what is the action this button does?

Reply:
This function just checks if the file has multiple sides or not. If nextSlideBtnEnabledSelector is found then the file has multiple slides otherwise not. Since this function is not being used in the current scenario, I am removing it.

Reply:
This button selects the option "Open in Presentation Viewer" from the menu containing various options like Open in Text Editor, Open in Presentation Viewer, Share, Copy link, Download and so on.

Reply:
This function generally checks if the "next slide" button exists or not. Since it is not used in this scenario, I removed this function for now.